docs/content/getting-started.md

```diff
@@ -51,3 +51,27 @@ export default const MyApp = () => (
 ```
 
 This will apply the same `color`, `font-family`, and `line-height` styles to the `<body>` as [Primer CSS's base styles](https://github.com/primer/css/blob/master/src/base/base.scss#L15-L20).
+
+## TypeScript
+
+Primer Components includes TypeScript support with no additional `@types` package necessary. You will need to install `@types/styled-components` and `@types/react` if you import either of those packages.
```

Comment:
Is this contradictory since both of those packages are required to get Primer Components to work 🤔

Reply:
IIRC if you don't import styled-components, you don't need the types (our library is all JS at this point).

Reply:
I updated this section to be more clear